types/leaflet/index.d.ts

```diff
@@ -728,10 +734,13 @@ export class LayerGroup<P = any> extends Layer {
     feature?: geojson.FeatureCollection<geojson.GeometryObject, P> | geojson.Feature<geojson.MultiPoint, P> | geojson.GeometryCollection;
 }
 
+// @factory L.layerGroup(layers?: Layer[], options?: Object)
```

Comment:
This seems to be a redundant jsdoc comment, especially with all these type definitons.